charmhelpers/contrib/openstack/utils.py

```diff
@@ -1841,3 +1841,34 @@ def series_upgrade_complete(resume_unit_helper=None, configs=None):
         configs.write_all()
         if resume_unit_helper:
             resume_unit_helper(configs)
+
+
+def run_in_apache(package, release=None):
```

Comment:
Hey David, I put some comments about this code in https://review.openstack.org/#/c/641003/1 if you can address those. Thanks.

Reply:
hey, these have been addressed